src/test/java/reposense/parser/ArgsParserTest.java

```diff
@@ -93,7 +97,7 @@ public void parse_help_throwsHelpScreenException() throws HelpScreenException, P
 
     @Test
     public void parse_allCorrectInputsAlias_success() throws ParseException, IOException, HelpScreenException {
-        String input = String.format("-c %s -o %s -s 01/07/2017 -u 30/11/2017 -f java adoc html css js -i -v",
+        String input = String.format("-c %s -o %s -s 01/07/2017 -u 30/11/2017 -f java adoc html css js -i -v -t UTC+08",
```

Comment:
for non-timezone unit tests, we can just use a standard value, e.g. `UTC+08`, and make it a constant.